JS-Drum-Kit/app.js

```diff
@@ -0,0 +1,32 @@
+// Play drum by keyboard press
+window.addEventListener('keydown' , playDrum);
```

Comment:
please move adding event listeners to the bottom of file

Reply:
done